### PR TITLE
NetSuite API: Mappings App Cleanup - Rename, Refactor, Remove, Documentation

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

File: apps/mappings/models.py [hunk 1]
```diff
@@ -45,10 +44,9 @@ class GeneralMapping(models.Model):
     vendor_payment_account_name = models.CharField(max_length=255, help_text='VendorPayment Account name', null=True)
     default_ccc_vendor_id = models.CharField(max_length=255, help_text='Default CCC Vendor ID', null=True)
     default_ccc_vendor_name = models.CharField(max_length=255, help_text='Default CCC Vendor Name', null=True)
-    workspace = models.ForeignKey(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
+    workspace = models.OneToOneField(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
     created_at = models.DateTimeField(auto_now_add=True, help_text='Created at datetime')
     updated_at = models.DateTimeField(auto_now=True, help_text='Updated at datetime')
 
     class Meta:
-        unique_together = ('location_name', 'workspace')
```
Comment (ashwin1111) [inline apps/mappings/models.py:53]: don't think this is needed

File: apps/mappings/models.py [hunk 2]
```diff
@@ -45,10 +44,9 @@ class GeneralMapping(models.Model):
     vendor_payment_account_name = models.CharField(max_length=255, help_text='VendorPayment Account name', null=True)
     default_ccc_vendor_id = models.CharField(max_length=255, help_text='Default CCC Vendor ID', null=True)
     default_ccc_vendor_name = models.CharField(max_length=255, help_text='Default CCC Vendor Name', null=True)
-    workspace = models.ForeignKey(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
+    workspace = models.OneToOneField(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
```
Comment (ashwin1111) [inline apps/mappings/models.py:47]: Changed relation

File: apps/mappings/models.py [hunk 3]
```diff
@@ -13,12 +13,11 @@ class SubsidiaryMapping(models.Model):
     id = models.AutoField(primary_key=True)
     subsidiary_name = models.CharField(max_length=255, help_text='NetSuite Subsidiary name')
     internal_id = models.CharField(max_length=255, help_text='NetSuite Subsidiary id')
-    workspace = models.ForeignKey(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
+    workspace = models.OneToOneField(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
```
Comment (ashwin1111) [inline apps/mappings/models.py:16]: Changed relation

File: apps/mappings/models.py [hunk 4]
```diff
@@ -13,12 +13,11 @@ class SubsidiaryMapping(models.Model):
...
     created_at = models.DateTimeField(auto_now_add=True, help_text='Created at datetime')
     updated_at = models.DateTimeField(auto_now=True, help_text='Updated at datetime')
 
     class Meta:
-        unique_together = ('subsidiary_name', 'workspace')
```
Comment (ashwin1111) [inline apps/mappings/models.py:21]: don't think this is needed

File: apps/mappings/utils.py [hunk 5]
```diff
@@ -1,142 +0,0 @@
...
-
-from apps.netsuite.tasks import schedule_vendor_payment_creation
-from apps.workspaces.models import Configuration
-from fyle_netsuite_api.utils import assert_valid
-
-from .models import GeneralMapping, SubsidiaryMapping
-from .tasks import schedule_auto_map_ccc_employees
-
-
-class MappingUtils:
```
Comment (ashwin1111) [inline apps/mappings/utils.py:16]: RIP `MappingUtils`

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**